test/test_modified_scripts.py

```diff
@@ -52,11 +46,8 @@ def get_modified_scripts():
         local_script, local_version = item.lower().split(",")
         # check for new scripts or a change in versions for present scripts
         # repo script versions compared with upstream.
-        if local_script not in upstream_versions.keys():
+        if LooseVersion(local_version) != upstream_versions[local_script]:
```

Comment:
Let's get this changed to `LooseVersion(local_version) > upstream_versions[local_script]`

To try and test if the local version is greater.

Reply:
Ok, would do so.

Reply:
I think if you change this over, we can merge this in!

Reply:
Hey I know this is merged in, can you get the LooseVersion change in the next PR?

Reply:
Definitely would do so.

Reply:
@harshitbansal05, Could you create and issue for this so that we do not forget